sections/header.liquid

```diff
@@ -89,6 +89,7 @@
   }
 
   .section-header {
+    transform: translate3d(0,0,0); /* This is for fixing a Safari z-index issue. PR #2147 */
```

Comment:
This actually brings in a new issue in Safari 😓 
So it seems like the original approach might be better 🤔 
[video](https://screenshot.click/15-39-nzdoj-o1bu3.mp4)

Reply:
Yeah, it's interesting I don't see this issue on my end but there is another one you didn't mentioned that I can see, the sticky header now works always (on scroll up and down) rather than works on scroll up only 😅

Huge thanks to Ludo for catching it!